### PR TITLE
MAISTRA-2251: Update SME status upon failures

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: go.mod
Comment (brian-avery) [inline go.mod]: what's the plan with this replace?

Comment (brian-avery) [reply go.mod]: (i.e. do you plan to remove once https://github.com/maistra/api/pull/14#pullrequestreview-679713002 is merged and before this merges?)

Comment (jwendell) [reply go.mod]: Sure. Once api is merged I will replace it here

Comment (brian-avery) [reply go.mod]: Code looks good to me. Will approve now pending the update for the two replaces.

**Review by brian-avery — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Pull request merged**
